apps/designer/app/canvas/shared/use-drag-drop.ts

```diff
@@ -167,12 +167,29 @@ export const useDragAndDrop = () => {
         return false;
       }
 
+      // When trying to drag an inline instance, drag its parent instead
+      if (components[instance.component].isInlineOnly) {
```

Comment:
I am realizing just now that isInlineOnly is a potentially confusing name, how would you call it?

Reply:
Depends on what you find confusing. I think there're two things. 

1. The word "only" suggest there can be components that can be used both for formatting and as standalone components.

2. "Inline" suggest this is about block / inline distinction, but this is more about "used for formatting" / "standalone" distinction.

Maybe `isFormatting`?

Reply:
maybe we should be just more explicit, e.g. since those are used to identify draggability, we can say: isDraggable false instead of isInline:true

Reply:
I was trying to describe what that component IS, but its hard, because this is more about how its being used

Reply:
isReparentable?

Reply:
Something drag&drop related like `isReparentable` will feel wrong here: https://github.com/webstudio-is/webstudio-designer/blob/94a827b9f460c18f50f00073552084d21e4f8000/apps/designer/app/canvas/shared/use-track-selected-element.ts#L102-L104